ting_openformat.field.inc

```diff
@@ -376,7 +379,49 @@ function _ting_openformat_parse_element($elements = NULL, $render_array = array(
   }
   $has_search_code = FALSE;
   foreach ($elements as $key => $value) {
-    if ( is_array($value) && array_key_exists('accessUrl', $value ) ) {
+    if ($key === 'accessInformationNote' && is_array($value)) {
+      foreach ($value as $k => $v) {
+        if (array_key_exists('accessUrl', $v )) {
+          $header = array(
+            '#theme' => 'ting_openformat_field',
+            '#openformat_header' =>  array('#markup' => check_plain($v['accessUrlDescriptor'])),
+          );
+          $render_array[] = array(
+            '#theme' => 'ting_openformat_field',
+            '#openformat_field' => array(
+              '#theme' => 'link',
+              '#text' => $v['accessUrlDisplay'],
+              '#path' => $v['accessUrl'],
+              '#options' => array(
+                'attributes' => array('target' => '_blank'),
+                'html' => TRUE,
+              ),
+            ),
+          );
+        }
+        else if (is_array($value)) {
```

Comment:
Du har jo i line 382 fastslået at $value er et array, hvorfor skal det gøres igen?

Reply:
Skulle det have været $v i stedet for $value?

Reply:
nej, det var vist et unødvendigt check